crate/operator/handlers/handle_update_cratedb.py

```diff
@@ -119,8 +120,28 @@ async def update_cratedb(
                     "size"
                 ) != new_spec.get("resources", {}).get("disk", {}).get("size"):
                     do_expand_volume = True
-
-    if not do_upgrade and not do_restart and not do_scale and not do_expand_volume:
+                elif (
```

Comment:
Can we perhaps simplify this `if` with some sort of helper function? Sure we could write some fancy for loop here :)